client/js/autocompletion.js

```diff
@@ -282,7 +282,7 @@ function completeCommands(word) {
 function completeChans(word) {
 	const words = [];
 
-	sidebar.find(".chan")
+	sidebar.find(".chan.active")
```

Comment:
This will only find the active channel, not network. You have to go up to the parent.

Reply:
I've now fixed it, and the commits are squashed into one as the Contributing guidelines say.

Reply:
Please change spaces to tabs on the lines that have been added. We use tabs in our codebase.

Reply:
I am sorry I missed that, I have also fixed that now.